scripts/library.py

```diff
@@ -309,23 +351,26 @@ def create_record_visual_data(bot, mot, tot, to_draw):
     tot (array): average treble volume over time
     to_draw (bool): controls whether to plot it or not
   """
+  
+  skip = 10 # Only process every 10th sample point (reduces density)
```

Comment:
Is there a big functional difference between skipping data points and just lowering the sampling frequency?